plans/management/commands/autorenew_accounts.py

```diff
@@ -0,0 +1,11 @@
+from django.core.management import BaseCommand
+from plans import tasks
+
+
+class Command(BaseCommand):
+    help = 'Autorenew accounts and with recurring payments'
```

Comment:
This could be marked for translation ?

Reply:
This is management command - this text will be outputed only for system administrators. I don't think, it should be translated, and I don't even think, that translation would work on that.
And also all of the current management commands are untranslated.

Or do you execute management command from other place than command line?